main.py

```diff
@@ -8,21 +9,31 @@
 from dotenv import load_dotenv
 
 load_dotenv()
+
 app = FastAPI()
-firebase_app = firebase_admin.initialize_app(
-    credential=None,
-    options={"storageBucket": os.getenv("CLOUD_STORAGE_BUCKET")}
-)
 
+# SSL certificates for HTTPS
+if os.getenv("ENV") == "prod":
```

Comment:
this way you don't need the SSL keys or certificates when developing locally. But when deploying to Google Cloud, I set the environment variable `ENV=prod`. So we can have an HTTPS url